### PR TITLE
Source Google Analytics Data API: Fix formatting

### PR DESCRIPTION
**Pull request by artem1205**
## What

Fix Source Google Analytics Data API Connector health report

## How
Fix formatting

## Recommended reading order
1. `expected_records.jsonl`

## 🚨 User Impact 🚨

no breaking changes

## Pre-merge Actions

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

### CODE REVIEW

**Comment by octavia-squidington-iii** [comment]
##  source-google-analytics-data-api test report (commit [`3b87c9f01a`](https://github.com/airbytehq/airbyte/pull/27531/commits/3b87c9f01aba35393dc879ffb0ee904d512d5a81)) - ❌

⏲️  Total pipeline duration: 1782 seconds

| Step                                                                                    | Result   |
|:----------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-analytics-data-api/metadata.yaml | ✅        |
| Connector version semver check.                                                         | ✅        |
| Connector version increment check.                                                      | ❌        |
| QA checks                                                                               | ❌        |
| Code format checks                                                                      | ✅        |
| Connector package install                                                               | ✅        |
| Build source-google-analytics-data-api docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                              | ✅        |
| Acceptance tests                                                                        | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5326923622)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-analytics-data-api test
```

**Comment by octavia-squidington-iii** [comment]
##  source-google-analytics-data-api test report (commit [`4be3ac02fc`](https://github.com/airbytehq/airbyte/pull/27531/commits/4be3ac02fcb477cf5ad42ccb77b38475f90acd8c)) - ❌

⏲️  Total pipeline duration: 1789 seconds

| Step                                                                                    | Result   |
|:----------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-analytics-data-api/metadata.yaml | ✅        |
| Connector version semver check.                                                         | ✅        |
| Connector version increment check.                                                      | ❌        |
| QA checks                                                                               | ❌        |
| Code format checks                                                                      | ✅        |
| Connector package install                                                               | ✅        |
| Build source-google-analytics-data-api docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                              | ✅        |
| Acceptance tests                                                                        | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5327702843)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-analytics-data-api test
```

File: airbyte-integrations/connectors/source-google-analytics-data-api/source_google_analytics_data_api/api_quota.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
 from .utils import API_LIMIT_PER_HOUR
 
+
```
Comment (davydov-d) [inline airbyte-integrations/connectors/source-google-analytics-data-api/source_google_analytics_data_api/api_quota.py:14]: code changes require a version bump as well

**Review by davydov-d — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-google-analytics-data-api/source_google_analytics_data_api/api_quota.py (hunk 1 above)
Comment (artem1205) [reply airbyte-integrations/connectors/source-google-analytics-data-api/source_google_analytics_data_api/api_quota.py:14]: updated

**Review by davydov-d — APPROVED** [review]

**Comment by octavia-squidington-iii** [comment]
##  source-google-analytics-data-api test report (commit [`414cfb4089`](https://github.com/airbytehq/airbyte/pull/27531/commits/414cfb40895e060c574ac1d18432e1fe2438a075)) - ✅

⏲️  Total pipeline duration: 1791 seconds

| Step                                                                                    | Result   |
|:----------------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-google-analytics-data-api/metadata.yaml | ✅        |
| Connector version semver check.                                                         | ✅        |
| Connector version increment check.                                                      | ✅        |
| QA checks                                                                               | ✅        |
| Code format checks                                                                      | ✅        |
| Connector package install                                                               | ✅        |
| Build source-google-analytics-data-api docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                              | ✅        |
| Acceptance tests                                                                        | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5332781613)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-google-analytics-data-api test
```

**Pull request merged**